package.json

```diff
@@ -69,5 +69,8 @@
     "packages": [
       "packages/*"
     ]
+  },
+  "dependencies": {
+    "logkitty": "^0.7.1"
```

Comment:
Im not sure this actually fixes it - if you look at yarn.lock,  logkitty@^0.6.0 is still installed and use separately.

Reply:
As for the fix, I updated the CLI following the steps here: https://github.com/react-native-community/cli#updating-the-cli